src/chttpd/src/chttpd_db.erl

```diff
@@ -1816,7 +1816,14 @@ parse_changes_query(Req) ->
         {"heartbeat", "true"} ->
             Args#changes_args{heartbeat=true};
         {"heartbeat", _} ->
-            Args#changes_args{heartbeat=list_to_integer(Value)};
+            try list_to_integer(Value) of
+                V when V > 0 ->
+                    Args#changes_args{heartbeat=heartbeat_integer};
```

Comment:
I don't think `heartbeat_integer` is correct.
```
Args#changes_args{heartbeat=heartbeat_integer};
```

Reply:
Good call! I've just pushed a commit to fix this.